src/Microsoft.Compatibility.AsyncInterfaces/ref/Microsoft.Compatibility.AsyncInterfaces.Forwards.cs

```diff
@@ -0,0 +1,10 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+[assembly: System.Runtime.CompilerServices.TypeForwardedTo(typeof(System.IAsyncDisposable))]
+[assembly: System.Runtime.CompilerServices.TypeForwardedTo(typeof(System.Collections.Generic.IAsyncEnumerable<>))]
+[assembly: System.Runtime.CompilerServices.TypeForwardedTo(typeof(System.Collections.Generic.IAsyncEnumerator<>))]
+[assembly: System.Runtime.CompilerServices.TypeForwardedTo(typeof(System.Runtime.CompilerServices.AsyncIteratorStateMachineAttribute))]
+[assembly: System.Runtime.CompilerServices.TypeForwardedTo(typeof(System.Runtime.CompilerServices.ConfiguredAsyncDisposable))]
+[assembly: System.Runtime.CompilerServices.TypeForwardedTo(typeof(System.Runtime.CompilerServices.ConfiguredCancelableAsyncEnumerable<>))]
```

Comment:
After this goes in, I'll submit a PR to add in the remaining few necessary types (which all require a few tweaks to target netstandard2.0).

Reply:
Sounds good. For each one that gets added we will need to run a `dotnet msbuild  src/Microsoft.Compatibility.AsyncInterfaces.csproj /t:GenerateReferenceSource /p:TargetGroup=netstandard` in order to update the surface area for netstandard.